bluepyopt/ephys/morphologies.py

```diff
@@ -180,60 +180,46 @@ def set_nseg(self, icell):
             section.nseg = 1 + 2 * int(section.L / self.nseg_frequency)
 
     @staticmethod
-    def replace_axon(sim=None, icell=None, axon_stump_length=60, n_sections=2):
+    def replace_axon(sim=None, icell=None,
+                     axon_stub_length=60, axon_nseg_frequency=40):
         """Replace axon"""
 
         nsec = len([sec for sec in icell.axonal])
 
-        print(f"In replace_axon {nsec = }")
-
         if nsec == 0:
-            ais_diams = [1] * n_sections
+            ais_diams = [1, 1]
         elif nsec == 1:
-            sec_x = [min(1, x/n_sections * axon_stump_length/icell.axon[0].L)
-                     for x in range(0, n_sections)]
-            ais_diams = [icell.axon[0](sx).diam for sx in sec_x]
+            ais_diams = [icell.axon[0].diam, icell.axon[0].diam]
         else:
-            ais_diams = [icell.axon[0].diam] * n_sections
+            ais_diams = [icell.axon[0].diam, icell.axon[0].diam]
             # Define origin of distance function
             sim.neuron.h.distance(0, 0.5, sec=icell.soma[0])
 
-            end_point_dist = np.arange(1, n_sections+1) \
-                * axon_stump_length / n_sections
-
-            idx = 1
-
             for section in icell.axonal:
-                if idx >= len(ais_diams):
+                # If distance to soma is larger than
+                # axon_stub_length, store diameter
+                if sim.neuron.h.distance(1, 0.5, sec=section) \
+                        > axon_stub_length:
+                    ais_diams[1] = section.diam
                     break
-                if sim.neuron.h.distance(1, 0.5, sec=section)\
-                        < end_point_dist[idx]:
-                    ais_diams[idx:] = section.diam
-                else:
-                    idx += 1
 
         for section in icell.axonal:
             sim.neuron.h.delete_section(sec=section)
 
         # Create new axon array
-        sim.neuron.h.execute(f"create axon[{n_sections}]", icell)
+        sim.neuron.h.execute('create axon[2]', icell)
 
         for index, section in enumerate(icell.axon):
-            section.nseg = 1
-            section.L = axon_stump_length / n_sections
+            section.L = axon_stub_length / 2
+            section.nseg = 1 + 2 * int(section.L/axon_nseg_frequency)
```

Comment:
Could you add spaces around the '/' -> section.L / axon_nseg_frequency
The linting test fails without the spaces around an operator

Reply:
Sure, added the spaces.